### PR TITLE
Updated worker documentation and added a few usage shortcuts

### PR DESCRIPTION
**Pull request by gardusig**

### CODE REVIEW

**Comment by manan164** [comment]
Hi @gardusig , One thing to note is, when we publish a new package, we have manually copy paste the ReadMe file in nuget.org. So the latest ReadMe is shown in the latest package.

**Review by manan164 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ using Conductor.Client.Authentication;
 
 var configuration = new Configuration() {
     BasePath = basePath,
-    AuthenticationSettings = new OrkesAuthenticationSettings(keyId, keySecret)
+    AuthenticationSettings = new OrkesAuthenticationSettings("keyId", "keySecret")
```
Comment (v1r3n) [inline README.md:44]: is this okay?

File: Tests/Util/ApiUtil.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ static ApiUtil()
         {
             _configuration = new Configuration()
             {
-                Timeout = 30000,
+                Timeout = 7500,
```
Comment (v1r3n) [inline Tests/Util/ApiUtil.cs:22]: why so low timeout?

Comment (gardusig) [reply Tests/Util/ApiUtil.cs:22]: There are quite a few retries with sleep in between, it made tests take too long to complete

**Pull request merged**